plugins/org/src/components/MembersTab/MembersTab.tsx

```diff
@@ -0,0 +1,141 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import Alert from '@material-ui/lab/Alert';
+import {
+  Box,
+  Card,
+  createStyles,
+  Grid,
+  Link,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { InfoCard, Progress, useApi } from '@backstage/core';
+import {
+  UserEntity,
+  RELATION_MEMBER_OF,
+  Entity,
+} from '@backstage/catalog-model';
+import { Link as RouterLink, generatePath } from 'react-router-dom';
+import { catalogApiRef, entityRouteParams } from '@backstage/plugin-catalog';
+import { useAsync } from 'react-use';
+import { Avatar } from '../Avatar';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    card: {
+      overflow: 'visible',
+      position: 'relative',
+    },
+  }),
+);
+
+const MemberComponent = ({
+  member,
+  entity,
+}: {
+  member: UserEntity;
+  entity: Entity;
+}) => {
+  const classes = useStyles();
+  const { name: metaName } = member.metadata;
+  const { profile } = member.spec;
+  return (
+    <Grid item md={3}>
+      <Card raised classes={{ root: classes.card }}>
+        <Box
+          display="flex"
+          flexDirection="column"
+          m={3}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <Avatar
+            displayName={profile?.displayName}
+            picture={profile?.picture}
+            customStyles={{
+              position: 'absolute',
+              top: '-25px',
+            }}
+          />
+          <Box py={4} textAlign="center">
+            <Typography variant="h5">
+              <Link
+                component={RouterLink}
+                to={generatePath(
+                  `/catalog/:namespace/user/${metaName}`,
+                  entityRouteParams(entity),
+                )}
+              >
+                {profile?.displayName}
+              </Link>
+            </Typography>
+            <Typography variant="caption">{profile?.email}</Typography>
+          </Box>
+        </Box>
+      </Card>
+    </Grid>
+  );
+};
+
+export const MembersTab = ({ entity }: { entity: Entity }) => {
+  const {
+    metadata: { name: groupName },
+  } = entity;
+  const catalogApi = useApi(catalogApiRef);
+
+  const { loading, error, value: members } = useAsync(async () => {
+    const membersList = await catalogApi.getEntities({
+      filter: {
+        kind: 'User',
+      },
+    });
+    const groupMembersList = ((membersList.items as unknown) as Array<
+      UserEntity
+    >).filter(member =>
+      member?.relations?.some(
+        r => r.type === RELATION_MEMBER_OF && r.target.name === groupName,
+      ),
+    );
+    return groupMembersList;
+  }, [catalogApi]);
+
+  if (loading) return <Progress />;
+  else if (error) return <Alert severity="error">{error.message}</Alert>;
```

Comment:
Not sure if this is a matter of taste, but I would prefer to always use braces for control statements.

Reply:
yep, for sure